cmd/connection_cleaner/main.go

```diff
@@ -33,18 +40,36 @@ func getRunningPods(dnsName string) (RunningPods, error) {
 	return runningPodMap, nil
 }
 
-func processConnection(dryRun bool, runningPods RunningPods, redisClient *redis.Client, account, nodeID, podName string) {
+func processConnection(dryRun bool, runningPods RunningPods, redisClient *redis.Client, account, nodeID, podName string, connCount *Metrics) {
 	if _, exists := runningPods[podName]; !exists {
 		fmt.Printf("Pod (%s) down!  This entry should be removed:  %s:%s\n", podName, account, nodeID)
 		if dryRun == false {
 			controller.UnregisterWithRedis(redisClient, account, nodeID, podName)
+			connCount.unregisterStaleConnectionFromRedis.Inc()
 		}
 	}
 }
 
+func pushValues(cfg *config.Config, connCount *Metrics) {
```

Comment:
nit-picky:  What do you think about naming this method something a little bit more descriptive?  Something like pushMetricsToPrometheus()?